src/styles/index.ts

```diff
@@ -1748,7 +1748,7 @@ const styles = (theme: ThemeColors) =>
 
         emojisWithTextFontSizeAligned: {
             fontSize: variables.fontSizeEmojisWithinText,
-            marginVertical: -7,
+            marginBottom: -7,
```

Comment:
Can you show a before/after of this change in terms of how it affects a non-Android platform? Just want to make sure we're still getting expected behavior.